config/config.go

```diff
@@ -1,16 +1,17 @@
 package config
 
 import (
-	"github.com/kelseyhightower/envconfig"
 	"time"
+
+	"github.com/kelseyhightower/envconfig"
 )
 
 // Config values for the application.
 type Config struct {
 	BindAddr                 string        `envconfig:"BIND_ADDR"`
 	KafkaAddr                []string      `envconfig:"KAFKA_ADDR"`
-	FilterJobConsumerGroup   string        `envconfig:"FILTER_JOB_CONSUMER_GROUP"`
-	FilterJobConsumerTopic   string        `envconfig:"FILTER_JOB_CONSUMER_TOPIC"`
+	FilterConsumerGroup      string        `envconfig:"FILTER_JOB_CONSUMER_GROUP"`
+	FilterConsumerTopic      string        `envconfig:"FILTER_JOB_CONSUMER_TOPIC"`
```

Comment:
Can the env variables be rename as well? The configuration files in dp-ci will also need updating and the repo's README.md

Reply:
I was thinking of doing this as a separate pull request. I didn't want to break cmd-develop due to environment configurations; when it could break for so many other reasons. So if you are happy with me doing this separately?

Reply:
Thats sounds fine 👍